src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -51,14 +51,57 @@
+ * against a set of known genotypes in the supplied genotype file (in VCF format).
  *
- * @author Tim Fennell
+ * <h4> Summary </h4>
+ * Computes a fingerprint from the supplied input file (SAM/BAM or VCF) file and
+ * compares it to the expected fingerprint genotypes provided. The key output is a LOD score
+ * which represents the relative likelihood of the sequence data originating from the same
+ * sample as the genotypes vs. from a random sample.  Two outputs are produced: (1) a summary
+ * metrics file that gives metrics of fingerprint match when comparing the input to a set of
+ * genotypes for the expected sample.  at the single sample level (if the input was a VCF) or at the read
+ * level (lane or index within a lane) (if the input was a SAM/BAM) and (2) a detail metrics file that
+ * contains an individual SNP/Haplotype comparison within a fingerprint comparison.  The metrics files
+ * fill the fields of the classes {@link FingerprintingSummaryMetrics} and {@link FingerprintingDetailMetrics}.
+ * The output files may be specified individually using the SUMMARY_OUTPUT and DETAIL_OUTPUT options.
+ * Alternatively the OUTPUT option may be used instead to give the base of the two output
+ * files, with the summary metrics having a file extension ".{@value CheckFingerprint#FINGERPRINT_SUMMARY_FILE_SUFFIX}",
+ * and the detail metrics having a file extension ".{@value CheckFingerprint#FINGERPRINT_DETAIL_FILE_SUFFIX}".
+ * <p>
+ * <h4>Example</h4>
+ * <h3>Comparing a bam against known genotypes:</h3>
+ * <pre>
+ *     java -jar picard.jar CheckFingerprint \\ <br />
+ *          INPUT=sample.bam \\ <br />
+ *          GENOTYPES=sample_genotypes.vcf <br />
+ *          HAPLOTYPE_DATABASE=fingerprinting_haplotype_database.txt \\ <br />
+ *          OUTPUT=sample_fingerprinting
+ * </pre>
+ * <p>
+ * <h4>Detailed Explanation</h4>
+ *
+ * This tool calculates a single number that reports the LOD score for identity check between the INPUT
+ * and the GENOTYPES. A positive value indicates that the data seems to have come from the same individual
+ * or, in other words the identity checks out. The scale is logarithmic (base 10), so a LOD of 6 indicates
+ * that it is 1,000,000 more likely that the data matches the genotypes than not. A negative value indicates
+ * that the data do not match. A score that is near zero is inconclusive and can result from low coverage
+ * or non-informative genotypes.
+ *
+ * The identity check makes use of haplotype blocks defined in the HAPLOTYPE_MAP file to enable it to have higher
+ * statistical power for detecting identity or swap by aggregating data from several SNPs in the haplotype block. This
+ * enables an identity check of samples with very low coverage (e.g. ~1x mean coverage).
+ *
+ * When provided a VCF, the identity check looks at the PL, GL and GT fields (in that order) and uses the first one that
+ * it finds.
+ *
+ *  @author Tim Fennell
  */
+
```

Comment:
Remove blank lines

Reply:
I  have a branch/PR in gatk that transforms the picard syntax to GATK in the GATKHelpDoclet so that GATKs docs on Picard tools have the right syntax as exposed by GATK.

Reply:
Then Picard repo can keep their syntax while GATK's web-docs shows the correct one for GATK.